### DISCUSSION TITLE
Wiki feature toelichting

### DISCUSSION THREAD

**Discussion by Serenoiia** [Matthew-Tangk/project-tech-A2#47] [Ideas]
Is het anders een plan om inplaats bij "Process" de info in een  "Per feature" pagina te maken in de wiki waar wij ons werk in zitten. 
Dan wordt process puur wat we die week hebben gedaan. 
En bij je feature kunnen we dan de werking uitleggen en onze code toelichten. (en de code daar neer zetten)

Klinkt dit als een goed idee?

**Comment by Matthew-Tangk** [comment]
Klinkt als een goed idee.

**Reply by Juul03** [reply]
Handig! Laten we dat doen!

**Reply by Serenoiia** [reply]
Ik heb de pagina's aangemaakt dus vul je vrij om ze aan te vullen!
